test/ref.test.js

```diff
@@ -1377,7 +1377,7 @@ test('Bad key', t => {
       })
       t.fail('Should throw')
     } catch (err) {
-      t.equal(err.message, 'Cannot find reference "extrenal#/definitions/projectId"')
+      t.equal(err.message, 'Schema with id "extrenal" is not found.')
```

Comment:
the error message is less specific

Reply:
not a problem to fix that

Reply:
https://github.com/fastify/json-schema-ref-resolver/pull/4